hdbt.theme

```diff
@@ -836,156 +662,94 @@ function hdbt_preprocess_paragraph(array &$variables): void {
 
   // Check if the paragraph is a type of hero, and it has with search as design.
   if (
-    $paragraph_type == 'hero' &&
     $paragraph->hasField('field_hero_design') &&
     $paragraph->get('field_hero_design')->value == 'with-search'
   ) {
 
     $search_urls = hdbt_get_global_url($variables['current_langcode']);
```

Comment:
Since `hdbt_preprocess` has been removed, current langcode does not get added into `$variables`, making calls to `hdbt_get_global_url` fail